src/detect_fig.py

```diff
@@ -0,0 +1,203 @@
+"""物体検出を行うモジュール.
+
+ベストショット画像を選択するための物体検出を行う。
+
+NOTE:
+    'learned_weight.pt'は以下にあります。
+    https://drive.google.com/drive/folders/1aUrB3Pj4hzt624SlJv8h1vVOBopAr48l
+    etrobocon2023-camera-system/yolo/の中にダウンロードしてください
+
+参考コード:
+    https://github.com/ultralytics/yolov5
+@author: kawanoichi
+"""
+
+import torch
+from pathlib import Path
+import os
+import numpy as np
+import sys
+from ultralytics.utils.plotting import Annotator, colors
+YOLO_PATH = os.path.join("..", "etrobocon2023-camera-system", "yolo")  # noqa
+sys.path.append(YOLO_PATH)  # noqa
+YOLO_PATH = Path(YOLO_PATH)  # noqa
+from models.common import DetectMultiBackend
+from utils.general import (
+    check_img_size, cv2, non_max_suppression, scale_boxes)
+from utils.torch_utils import select_device
+from utils.augmentations import letterbox
+
+IMAGE_PATH = os.path.join("..", "etrobocon2023-camera-system", "fig_image")  # noqa
+IMAGE_PATH = Path(IMAGE_PATH)  # noqa
+
+
+def check_exists(path) -> None:
```

Comment:
指定したパスのファイルが見つからない場合に、処理を分けたいこともあると思うから、bool値返した方がいいと思う。

その場合、関数名もtrue/falseがどっちかわかりやすい名前にしてほしい。

例えば
```suggestion
def is_exist(path) -> bool:
```